polling_stations/templates/fragments/help_your_council.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}
+<div class="card">
+    {% blocktrans with council.electoral_services_phone_numbers.0 as council_phone and council.name as council_name and council.electoral_services_email as council_electoral_services_email %}
+        <h3>Your election (might) need you!</h3>
+        <p>
+            A lot of people are needed to make an election happen.
+	    You can contact your council electoral services team to see if they need help.
+        </p>
+        <p>
+            You can <strong>
+		    <a href="{{ council_electoral_services_email }}">email</a>
+	    </strong> {{ council_name }} electoral services on or call them on
+	    <strong><a href="tel:{{ council_phone }}">{{ council_phone }}</a></strong>.
```

Comment:
@pmk01 do you have any thoughts on making this more explicit? I didn't know exactly what we're asking for. I also worry about pushing folk to ringing ES teams who may have enough on their plates already.

Reply:
I think we could make it more explicit, yes - 70+% of electoral admins say they don't have enough staff or resources for these elections, and every council site I have looked at is advertising for poll workers. To quote Martyn H when I put the suggestion to him: "Yes I think that would be a great idea- perhaps mention polling and counting staff, with multi day counts we’re all after bodies."

So I would suggest something like:

"A lot of people are needed to make an election happen. Your council may be hiring polling station and counting staff for the upcoming elections. Contact your council electoral services team to see if they need help."

I think any phone calls we create this way will be a drop in the ocean compared to their usual numbers, and offers of help will probably be welcome!

Reply:
Or a link to the webpage, as you suggested on the call!

Reply:
The websites we have are just council websites - so not actually a useful link. As you pointed out volume isn't going to be overwhelming so will leave the contact details in.